Src/GoogleApis.Auth.DotNet4/OAuth2/ServiceAccountCredential.cs

```diff
@@ -177,6 +191,16 @@ private static string CreateSerializedHeader()
             return NewtonsoftJsonSerializer.Instance.Serialize(header);
         }
 
+        /// <summary>Converts the PKCS8 PrivateKey to RSA Parameters. Uses the Bouncy Castle library.</summary>
+        private static RSAParameters ConvertPKCS8ToRSAParameters(string pkcs8PrivateKey)

```

Comment:
I'd still say these tests would be useful, but considering the corresponding test project doesn't exist yet and you are about to introduce it in the next pull request, is probably fine to leave this untested for now and have this tested along with DefaultCredentials